test/e2e/specs/site-editor/command-center.spec.js

```diff
@@ -17,36 +17,30 @@ test.describe( 'Site editor command center', () => {
 		await admin.visitSiteEditor();
 	} );
 
-	test.skip( 'Open the command center and navigate to the page create page', async ( {
+	test( 'Open the command center and navigate to the page create page', async ( {
 		page,
 	} ) => {
+		await page
+			.getByRole( 'button', { name: 'Open command center' } )
+			.focus();
 		await page.keyboard.press( 'Meta+k' );
-		const newPageButton = page.locator(
-			'role=option[name="Create a new page"i]'
-		);
-		await expect( newPageButton ).toBeVisible();
-
-		// Type a random post title
-		await page.keyboard.type( 'E2E Test Post' );
-		await page.click(
-			'role=option[name="Create a new post \\"E2E Test Post\\""i]'
-		);
-
+		await page.keyboard.type( 'new page' );
+		await page.getByRole( 'option', { name: 'Add new page' } ).click();
 		await page.waitForSelector( 'iframe[name="editor-canvas"]' );
 		const frame = page.frame( 'editor-canvas' );
```

Comment:
Why not `editor.canvas`?

Also, the `waitForSelector` call above isn't required.

Reply:
In that case, `.frameLocator( '[name="editor-canvas"]' )` might be better. Since we're navigating to a new page, maybe adding an assertion for the navigated url will also be good?

Reply:
The assertion of the "add title" cover the url change though.

Reply:
How? It can be any textbox with the same name on the same page if we don't check for navigation. It's also less readable what that means for test readers.

Reply:
Nvm if the functionality is subject to change though!

Reply:
The functionality is not subject to change but I'm still wondering whether checking the actual page content is. better than a URL which is more of a "technical" thing.

Reply:
End users can see and change the urls so I don't think it's a "technical" thing. Even if the actual url is not important, the _navigation_ itself might be worth asserting because that's an important part of the user flow IMO. It also helps the readers reading this test to understand that a navigation happens after clicking on the option.

Reply:
Safari and Chrome both hide the URLs recently. So I still think the content is actually a better thing to check and the URL is merely an implementation detail.

That said, I've opened this follow-up PR as auto merge triggered https://github.com/WordPress/gutenberg/pull/50835